examples/demo/site/modules/google_search_20/config/google_search_20.json

```diff
@@ -0,0 +1,5 @@
+{
+    "gcse": {
+     		"cx":""
```

Comment:
It is not obvious what should be set here ..
either add info to the README, or in this json file, add either comments, or meaningful field name.